File: TaskCheck.Application/Users/Commands/Add/AddUserCommand.cs
using MediatR;
using TaskCheck.Application.Users.DTO;
using TaskCheck.Domain.Shared;

namespace TaskCheck.Application.Users.Commands.Add;

 public record AddUserCommand (
    string Username,
    string FirstName,
    string Email,
    string Password,
    string PasswordRepeat) : IRequest<Result<RegisterResponse>>;

File: src/main/java/com/pr0f1t/taskcheck/commands/user/add/AddUserCommand.java
package com.pr0f1t.taskcheck.commands.user.add;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@AllArgsConstructor
@NoArgsConstructor
public class AddUserCommand{

    private String username;

    private String password;

    private String email;

}
